psi4/driver/qcdb/dashparam.py

```diff
@@ -309,6 +310,26 @@ def get_dispersion_aliases():
         'pbe0'        : {'s6': 1.000, 's8': 0.528823, 'a1': 0.007912, 'a2': 6.162326},
         'lcwpbe'      : {'s6': 1.000, 's8': 0.906564, 'a1': 0.563761, 'a2': 3.593680},
     },
+    'nl': { # TODO: citations
```

Comment:
I am wondering about `-rVV10`: you mentioned there is a third parameter which is the only difference between `-V` and `-rV` in eg. B97M-[r]V?

Reply:
The vv10 kernel is simplified, the external parameters remain the same i think. Would need to check.